src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -81,6 +81,7 @@ static Microsoft.CodeAnalysis.FindSymbols.SymbolFinder.FindSourceDeclarationsWit
 static Microsoft.CodeAnalysis.FindSymbols.SymbolFinder.FindSourceDeclarationsWithPatternAsync(Microsoft.CodeAnalysis.Project project, string pattern, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.ISymbol>>
 static Microsoft.CodeAnalysis.FindSymbols.SymbolFinder.FindSourceDeclarationsWithPatternAsync(Microsoft.CodeAnalysis.Solution solution, string pattern, Microsoft.CodeAnalysis.SymbolFilter filter, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.ISymbol>>
 static Microsoft.CodeAnalysis.FindSymbols.SymbolFinder.FindSourceDeclarationsWithPatternAsync(Microsoft.CodeAnalysis.Solution solution, string pattern, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.ISymbol>>
+static Microsoft.CodeAnalysis.Rename.RenameOptions.RenameFile.get -> Microsoft.CodeAnalysis.Options.Option<bool>
```

Comment:
@jasonmalinowski 	may know more?

Reply:
We should review it. Looks good to me? :-)

Reply:
(We don't really have a formal process that must be followed; in this case since this is following a well-established pattern I don't think there's too many concerns here, but we should at least make sure we document what the option means so we understand it ourselves.)

Reply:
Is there a specific place to document that? I didn't see documentation for the others

Reply:
Not really. "Documentation" in my case here just means use some XML doc comments so there's _something_.

Reply:
Actually I have concerns about how this API works, since it's forcing the rename to happen even if the original file name didn't match, and there's no way as a generic API caller to get the same logic. I'd propose keeping the option internal until we sort that out.